plugins/certs/commands

```diff
@@ -93,14 +93,14 @@ case "$1" in
 
       mkdir -p "$APP_SSL_PATH"
       dokku_log_info1 "Installing certificate and key..."
-      mv -f $TMP_WORK_DIR/server.key $TMP_WORK_DIR/server.crt $APP_SSL_PATH
+      mv -f $TMP_WORK_DIR/server.key $TMP_WORK_DIR/server.crt $TMP_WORK_DIR/server.csr $APP_SSL_PATH
       chmod 750 $APP_SSL_PATH
       chmod 640 $APP_SSL_PATH/server.key $APP_SSL_PATH/server.crt $APP_SSL_PATH/server.csr
       [[ -n "$DOMAIN" ]] && (dokku domains:add $APP $DOMAIN || nginx_build_config $APP)
       dokku_log_info1 "The following is a certificate signing request that can be used"
       dokku_log_info1 "to generate an 'officially' signed SSL certificate for $APP at $DOMAIN"
       dokku_log_info1 "by a CA of your choosing."
-      cat server.csr
+      cat "$APP_SSL_PATH/server.csr"
```

Comment:
Derp on me here I think. #1732 changes this behavior and fixes this bug. There's no real reason to keep the csr around IMO.